### PR TITLE
Add ADR for file export workflow

### PR DESCRIPTION
**Pull request by suzannehamilton**
Explain how the antivirus, checksum and file format ID tasks are run when a file is uploaded.

### CODE REVIEW

File: architecture-decision-records/0013-file-check-queues-and-lambdas.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+decided against them because when we tried [using them in the Alpha
+prototype][alpha-step-functions], we found that they failed when we hit the
+Lambda concurrent usage limit. This is currently 1000 functions, and we can hit
+this limit if a user uploads a consignment of thousands of small files. We don't
+have this problem with SQS, because the messages wait in the queues until AWS
+lets us start another Lambda.
+
+The initial step of downloading the file from S3 to Lambda is necessary for the
+antivirus and file format ID tasks, which need to access the file as if it was
+on disk.
```
Comment (MancunianSam) [inline architecture-decision-records/0013-file-check-queues-and-lambdas.md:79]: This is true but maybe put something to say that we're downloading to EFS rather than using the lambda local disk storage because the lambda disk storage is limited to 512Mb and EFS is effectively unlimited.
I'm not sure if this belongs in this section but it might be worth putting in something to say that the downside of EFS is that as its a network drive, disk access is much slower than using lambda local storage.

Also, as we're only downloading the files once, it speeds up the other lambdas as they don't have to download the same file individually, it simplifies the code a bit and it saves us on data transfer costs.

File: architecture-decision-records/0013-file-check-queues-and-lambdas.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+When a user uploads a folder to a TDR consignment, TDR needs to run a set of
+checks on each of those files. For the MVP release, the checks are:
+
+- [Antivirus scan]
+- [SHA-256 checksum], which can be compared to the checksum calculated in the
+  user's browser
+- [File format ID], which can be used to highlight files that The National
+  Archives does not accept, like password-protected files, or files a user might
+  have uploaded by mistake, like executable files
+
```
Comment (MancunianSam) [inline architecture-decision-records/0013-file-check-queues-and-lambdas.md:16]: Is it worth saying that the file format ID is not only checking for invalid files and that we need the file format id for when it leaves TDR? Or are we not bothered because those uses are outside the scope of TDR? I don't mind either way.

File: architecture-decision-records/0013-file-check-queues-and-lambdas.md (hunk 1 above)
Comment (suzannehamilton) [reply architecture-decision-records/0013-file-check-queues-and-lambdas.md]: Those are good points, thanks. I've added them. 👍

File: architecture-decision-records/0013-file-check-queues-and-lambdas.md (hunk 2 above)
Comment (suzannehamilton) [reply architecture-decision-records/0013-file-check-queues-and-lambdas.md:17]: Yep - I've added that.

**Review by MancunianSam — APPROVED** [review]
Looks good now, thanks :+1:
